app.js

```diff
@@ -0,0 +1,21 @@
+const taskManager = () => {
+  const inputTodo = document.getElementById("inputTodo");
+  const addTodoButton = document.getElementById("addTodoButton");
+  const listTodo = document.getElementById("listTodo");
+  const addTodo = () => {
```

Comment:
As suggested on the file review, this function will be responsible for delegating work
```suggestion
  const onAddTodo = () => {
```

Reply:
function naming updated